storage/src/mmr/iterator.rs

```diff
@@ -112,6 +112,38 @@ pub(crate) fn nodes_needing_parents(peak_iterator: PeakIterator) -> Vec<u64> {
     peaks
 }
 
+/// Returns the leaf's number based on its given position `leaf_pos`.
+///
+/// This computation is O(log2(n)) in the given position.
+#[allow(dead_code)] // remove this when we start using it
```

Comment:
nit: would you mind adding a`TODO:` prefix here (I use that to search for outstanding items)?

Reply:
done